### PR TITLE
Add privilege hierarchy images

### PR DESCRIPTION
**Pull request by Hunterness**
Uses the images introduced in https://github.com/neo-technology/neo4j-manual-modeling/pull/1620

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 1]
```diff
@@ -58,6 +53,8 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
     section("Database administration", "administration-security-administration-database-privileges", "enterprise-edition") {
       synopsis("This section explains how to use Cypher to manage privileges for Neo4j database administrative rights.")
       p("include::database/admin-role-database.asciidoc[]")
+      p("image::privilege-hierarchy-database.png[title=\"Database privileges hierarchy\"]")
```
Comment (mnd999) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala:57]: Again as for 4.3 you might want to add a little text here to separate it. As it's not sandwiched between two tables as in 4.3, it looks better.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 2]
```diff
@@ -285,6 +282,8 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
 
       section("The dbms `ROLE MANAGEMENT` privileges", "administration-security-administration-dbms-privileges-role-management", "enterprise-edition") {
         p("The dbms privileges for role management are assignable using Cypher administrative commands. They can be granted, denied and revoked like other privileges.")
+        p("image::privilege-hierarchy-dbms.png[title=\"Role management privileges hierarchy\"]")
```
Comment (mnd999) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala:286]: This one looks fine to me.

**Review by mnd999 — APPROVED** [review]
Looks good

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala (hunk 1 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala:57]: I did?

**Review by AlexicaWright — APPROVED** [review]
👍

**Pull request merged**